core/templates/domain/topic/subtopic-page.model.ts

```diff
@@ -31,15 +31,22 @@ export interface SubtopicPageBackendDict {
 }
 
 export class SubtopicPage {
+  // When creating a subtopicPage, property below are always
+  // initialized with null values. These are null until populated
+  // from the backend and are not null afterwards.
   constructor(
-    private id: string,
-    private topicId: string,
-    private pageContents: SubtopicPageContents,
+    private id: string | null,
+    private topicId: string | null,
+    private pageContents: SubtopicPageContents | null,
     private languageCode: string,
   ) {}
 
+  // Some methods have either string or null return value,
+  // because when we create interstitial subtopicpage
+  // their fields get null value.
```

Comment:
Make similar changes to this comment as I suggested above.

Reply:
Done